DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/L1/test_l1_autodetect.py

```diff
@@ -131,6 +131,7 @@ async def test_l1_autodetect(testbed, speed, duplex):
             await tgen_utils_start_traffic(tgen_dev)
             await asyncio.sleep(1)
             await tgen_utils_stop_traffic(tgen_dev)
+            await asyncio.sleep(5)
```

Comment:
can we check a state like traffic stooped instead of hardcoded wait ?
also if we are to go hardcoded , usually we need 3 pooling intervals for stats that is usually 6sec

Reply:
I think that this should be part of the ixnetwork framework. When I call ixnet.Traffic.Stop() I expect that the traffic should be stopped. I don't expect that I should be waiting another X seconds.